src/Config/ObjectModel/CliBool.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    public enum CliBool
```

Comment:
need comment describing why we need this enum versus all the other bool types in `InitOptions.cs`